### ISSUE TITLE
Finish integrating BLS into CDT main branch

### ISSUE THREAD

**Issue by arhag** [AntelopeIO/cdt#202]
https://github.com/AntelopeIO/cdt/pull/154 was merged into the `bls_integration` branch as part of completing the BLS milestone of IF. However, some changes still remain before the code can be merged into main.

First, the branch should be updated to use the leap-dev package generated from the `bls_integration` branch in Leap so that the integration tests can correctly run. Use this to ensure all the tests are running as intended.

Then, when https://github.com/AntelopeIO/leap/issues/1413 is completed, which means BLS support is integrated into the `main` branch of Leap, the `bls_integration` branch should again be updated to use the leap-dev package from the `main` branch of Leap. The code changes of that branch (with CICD all passing) should then be merged into the `main` branch of CDT.
```[tasklist]
### Tasks
- [ ] https://github.com/AntelopeIO/leap/issues/1462
```

**Comment by dimas1185** [comment]
integration tests do not work because of [leap#1462](https://github.com/AntelopeIO/leap/issues/1462)
make sense to finish after those will be fixed

**Comment by BenjaminGormanPMP** [comment]
EOS EVM PR in progress to support/unblock. **https://github.com/AntelopeIO/bls12-381/pull/5**

**Comment by ericpassmore** [comment]
Key dependency the [EOS EVM PR is closed](https://github.com/AntelopeIO/bls12-381/pull/5) closing this issue as well.
